.build/config/cassandra_ci.yaml

```diff
@@ -0,0 +1,355 @@
+      - dtest-upgrade-large
+      - long-test
+      - cqlsh-test
+
+  # The post-commit pipeline is a larger set of tests that include all supported JDKs.
+  # We expect different JDKs and variations on test suites to fail very rarely.
+  #
+  # Failures in these tests will be made visible on JIRA tickets shortly after
+  # test run on reference CI and committers are expected to prioritize
+  # rectifying any failures introduced by their work.
+  - name: post-commit
+    jdk:
+      - 11
+      - 17
+    jobs:
+      - unit
+      - unit-cdc
+      - compression
+      - test-oa
+      - test-system-keyspace-directory
+      - test-tries
+      - jvm-dtest
+      - jvm-dtest-upgrade
+      - dtest
+      - dtest-novnode
+      - dtest-offheap
+      - dtest-large
+      - dtest-large-novnode
+      - dtest-upgrade
+      - dtest-upgrade-large
+      - long-test
+      - cqlsh-test
+
+  # These are longer-term, much more rarely changing pieces of infrastructure or
+  # testing. We expect these to fail even more rarely than post-commit.
+  - name: nightly
+    jdk:
+      - 11
+      - 17
+    jobs:
+      - stress-test
+      - fqltool-test
+      - test-burn
+
+#-----------------------------------------------------------------------------
+# RESOURCE LIMITS, ALIASES, AND DEFAULT ENV VARS
+#-----------------------------------------------------------------------------
+# Downstream test orchestration needs to use <= the following values when running tests.
+# Increasing these values indicates  a change in resource allocation https://ci-cassandra.apache.org/ and should not be done downstream.
+small_executor: &small_executor {cpu: 4, memory: 1g, storage: 5g}
+medium_executor: &medium_executor {cpu: 4, memory: 6g, storage: 25g}
+large_executor: &large_executor {cpu: 4, memory: 16g, storage: 50g}
+
+# On test addition or change, we repeat the job many times to try and suss out flakes. Instead of having it be bespoke
+# per job, we want to provide some general guidelines for folks to default to and provide guidance on each test suite.
+repeat_default: &repeat_many 500
+repeat_less: &repeat_moderate 100
+repeat_tiny: &repeat_few 25
+
+# Default to at least one split
+default_split_num: &default_split_num let NUM_SPLITS=$(( $(wc -l < "$TEST_LIST_FILE") / $SPLIT_SIZE )); if [ "$NUM_SPLITS" -eq 0 ]; then NUM_SPLITS=1; fi
+
+# These env vars are required for tests to complete successfully given the run: commands, however downstream implementations
+# are welcome to change them as needed to setup their env
+default_env_vars: &default_env_vars
+  ANT_HOME: /usr/share/ant
+  KEEP_TEST_DIR: true
+  CASSANDRA_DIR: /home/cassandra/cassandra
+  CASSANDRA_DTEST_DIR: /home/cassandra/cassandra-dtest
+  CCM_CONFIG_DIR: ${DIST_DIR}/.ccm
+  TMPDIR: "$(mktemp -d)"
+  DIST_DIR: "${CASSANDRA_DIR}/build"
+  # Default to test.timeout as found in build.xml; should parse out of there in building local env and set this env var based on job
+  TEST_TIMEOUT: 480000
```

Comment:
this varies for each test type

Reply:
Resolved in current diff. using `env.TEST_TIMEOUT` as k/v pair to indicate string of which timeout to pull from build.xml.